libraries/Microsoft.Bot.Builder.Ai.QnA/QnAMakerMiddleware.cs

```diff
@@ -29,6 +32,14 @@ public QnAMakerMiddleware(QnAMakerMiddlewareOptions options, HttpClient httpClie
                 if (!string.IsNullOrEmpty(messageActivity.Text))
                 {
                     var results = await _qnaMaker.GetAnswers(messageActivity.Text.Trim()).ConfigureAwait(false);
```

Comment:
what if results returns null here? The trace isn't very interesting then.

Reply:
@mgbennet Tracing and Error Handling code should not throw errors. I would try really hard to avoid that.

Reply:
Fair point. I've added a null check around the trace code.